### PR TITLE
Wrong Dapr runtime version on About menu 

### PR DESCRIPTION
**Pull request by hueifeng**
fix: #201

### CODE REVIEW

File: pkg/version/version.go [hunk 1]
```diff
@@ -13,10 +13,20 @@ limitations under the License.
...
 var version = "edge"
 
 // GetVersion returns the current dashboard version
 func GetVersion() string {
 	return version
 }
+
+// GetRuntimeVersion returns the current runtime version
+func GetRuntimeVersion() string {
+	return strings.ReplaceAll(standalone.GetRuntimeVersion(), "\n", "")
```
Comment (artursouza) [inline pkg/version/version.go:31]: Does this work on K8s?

Comment (hueifeng) [reply pkg/version/version.go:31]: Let me fix it.

Comment (hueifeng) [reply pkg/version/version.go]: Don't merge, Let me fix that.

**Comment by hueifeng** [comment]
ping

**Comment by yaron2** [comment]
Did you test this on k8s?

**Comment by hueifeng** [comment]
> Did you test this on k8s?

Yes, I released the mirror image, for which I tested the effect of k8s.
https://hub.docker.com/r/hueifeng/dashboard

**Comment by hueifeng** [comment]
```
./build_kubernetes.sh hueifeng/dashboard:test dapr-system
```

`test_dashboard.yaml`  file added `      serviceAccountName: dashboard-reader`
```
apiVersion: apps/v1
kind: Deployment
metadata:
  name: dapr-dashboard
  labels:
    app: dapr-dashboard
spec:
  replicas: 1
  selector:
    matchLabels:
      app: dapr-dashboard
  template:
    metadata:
      labels:
        app: dapr-dashboard
    spec:
      serviceAccountName: dashboard-reader
      containers:
      - name: dapr-dashboard
        image: hueifeng/dashboard:test
        ports:
        - containerPort: 8080
        imagePullPolicy: Always
```

image

**Review by yaron2 — APPROVED** [review]

**Pull request merged**